src/pages/index.js

```diff
@@ -1,36 +1,69 @@
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg"
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg"
-  },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg"
-  },
-];
+import { enableValidation, resetValidation, settings } from "../scripts/validation.js";
+import Api from "../utils/api.js";
+import { setButtonText, resetForms } from "../utils/helper.js";
 
+
+// DOM Elements
+//Universal elements
+const closeButtons = document.querySelectorAll('.modal__close-btn')
+
+// Edit Profile Elements
+const avatarModalBtn = document.querySelector(".profile__avatar-btn");
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarFormElement = document.forms["avatar-form"];
+const avatarLinkInput = avatarModal.querySelector("#profile-avatar-input");
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const profileImage = document.querySelector(".profile__avatar")
+const editModal = document.querySelector("#edit-modal");
+const editFormElement = document.forms["profile-form"];
+const editModalNameInput = editModal.querySelector("#profile-name-input");
+const editModalDescriptionInput = editModal.querySelector("#profile-description-input");
+
+// Preview Elements
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+
+// Card-related Elements
+const cardModal = document.querySelector("#add-card-modal");
+const cardModalBtn = document.querySelector(".profile__add-btn");
+const cardFormElement = document.forms["add-card-form"];
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list")
+let selectedCard, selectedCardId;
+
+// Delete Elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteFormElement = document.forms["delete-form"];
+const cancelModalBtn = document.querySelector(".modal__cancel-btn")
+
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "1aeb74b8-8c69-4315-8442-0245d829c747",
+    "Content-Type": "application/json"
+  }
+});
+
+
+api.getAppInfo()
+  .then(([cards, userInfo]) => {
```

Comment:
* `Promise.all` is used for 2 or more promises

`userInfo` will be available here

```js

api
  .getAppInfo()
  .then(([cards, userInfo]) => {
    cards.forEach((item) => {
      renderCard(item);
    });
    profileName.textContent = userInfo.name;
    profileDescription.textContent = userInfo.about;
    profileImage.src = userInfo.avatar;
  })
  .catch(console.error);
```

Reply:
I see this issue I was basically calling getUserInfo twice. I have updated this and reflected  changes